haystack/retriever/dense.py

```diff
@@ -27,84 +27,63 @@ class DensePassageRetriever(BaseRetriever):
-                 use_amp: str = None,
-                 embed_title: bool = True
+                 embed_title: bool = True,
+                 remove_sep_tok_from_untitled_passages: bool = True
                  ):
         """
         Init the Retriever incl. the two encoder models from a local or remote model checkpoint.
-        The checkpoint format matches the one of the original author's in the repository (https://github.com/facebookresearch/DPR)
-        See their readme for manual download instructions: https://github.com/facebookresearch/DPR#resources--data-formats
+        The checkpoint format matches huggingface transformers' model format
 
         :Example:
 
             # remote model from FAIR
-            >>> DensePassageRetriever(document_store=your_doc_store, embedding_model="dpr-bert-base-nq", use_gpu=True)
+            >>> DensePassageRetriever(document_store=your_doc_store,
+                                      query_embedding_model="facebook/dpr-question_encoder-single-nq-base",
+                                      passage_embedding_model="facebook/dpr-ctx_encoder-single-nq-base",
+                                      use_gpu=True)
             # or from local path
-            >>> DensePassageRetriever(document_store=your_doc_store, embedding_model="some_path/ber-base-encoder.cp", use_gpu=True)
-
+            >>> DensePassageRetriever(document_store=your_doc_store,
+                                      query_embedding_model="local-path/query-checkpoint",
+                                      passage_embedding_model="local-path/ctx-checkpoint",
+                                      use_gpu=True)
         :param document_store: An instance of DocumentStore from which to retrieve documents.
-        :param embedding_model: Local path or remote name of model checkpoint. The format equals the 
-                                one used by original author's in https://github.com/facebookresearch/DPR. 
-                                Currently available remote names: "dpr-bert-base-nq" 
+        :param query_embedding_model: Local path or remote name of question encoder checkpoint. The format equals the
+                                one used by hugging-face transformers' modelhub models
+                                Currently available remote names: "facebook/dpr-question_encoder-single-nq-base"
+        :param passage_embedding_model: Local path or remote name of passage encoder checkpoint. The format equals the
+                                one used by hugging-face transformers' modelhub models
+                                Currently available remote names: "facebook/dpr-ctx_encoder-single-nq-base"
+        :param max_seq_len: Longest length of each sequence
         :param use_gpu: Whether to use gpu or not
         :param batch_size: Number of questions or passages to encode at once
         :param do_lower_case: Whether to lower case the text input in the tokenizer
-        :param encoder_model_type: 
```

Comment:
docstring missing for embed_title and remove_sep_tok_from_untitled_passages

Reply:
added the parameters to docstrings